### ISSUE TITLE
Some Babel errors are not caught by eslint plugin

### ISSUE THREAD

**Issue by mik01aj** [yannickcr/eslint-plugin-react#318]
Example:

```
ERROR in ./app/scripts/mobilePages/BookPage/index.jsx
Module build failed: SyntaxError: ......./BookPage/index.jsx: Line 20: Namespace tags are not supported. ReactJSX is not XML.
  18 |                     <button type="button" className="btn btn-default" data-toggle="collapse" data-target="#bs-example-navbar-collapse-2">
  19 |                       <span className="sr-only">Toggle navigation</span>
> 20 |                       <svg className="icon-three-bars"><use xlink:href="#icon-three-bars" /></svg>
     |                                                             ^
  21 |                     </button>
  22 |                     <button type="button" className="btn btn-default" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
  23 |                       <svg className="icon-search"><use xlink:href="#icon-search" /></svg>
```

In this case, I should have used `xlinkHref`.

I suppose these errors should be reported by the plugin too.

**Comment by mik01aj** [comment]
:+1: Thanks!
